Automattic/jetpack
# Top Posts and Pages Block: Tidy Up Titles

Fixes #34505

## Proposed changes:

Tidies up titles for the Top Posts and Pages block:

- Ensures that they're on separate lines
- Adds wrapping onto the next line
- Limits their length 

cc @kangzj 

### Other information:


## Jetpack product discussion


See original issue.

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:
With Beta blocks enabled, insert a Top Post & Pages block, then create some posts with really long titles. They should wrap appropriately here, and their length should be limited. Also, if you now only display the post title and category, they should remain on two different lines.

**Before:**
<img width="783" alt="Screenshot 2023-12-07 at 08 32 46" src="https://github.com/Automattic/jetpack/assets/43215253/8df5649e-a13d-4e3e-baa3-036348789f1a">

**After:**
<img width="682" alt="Screenshot 2023-12-07 at 08 31 43" src="https://github.com/Automattic/jetpack/assets/43215253/24c66c3a-5812-4a23-ba11-283c214ccea5">
